.github/workflows/label-check-ci.yml

```diff
@@ -10,7 +10,7 @@ on:
     # effect the labels because the merge requirements need checks to pass against the most recent
     # commit.
     types: [ opened, synchronize, reopened, labeled, unlabeled ]
-    branches: [ main ]
+    branches: [ 'main', 'rc/v*' ]
```

Comment:
I wonder:

Should we also update `create-docs-issues.yml` _or_ remove `rc/v*` branch here and `doc-labels` as required check for `rc/v*`?

Reply:
My reasoning was that it's in a patch release, theoretically we shouldn't require new release notes/docs for it... but we may as well use the same flow, so I'll update the `create-docs-issues.yml` here.